src/index.css

```diff
@@ -0,0 +1,742 @@
+
+  opacity: 0;
+  transition: opacity 1s;
+}
+
+#navbar .tooltip:hover .tooltiptext {
+  visibility: visible;
+  opacity: 1;
+}
+
+#navbar .avatar-large {
+  width: 3rem;
+  height: 3.2rem;
+}
+
+#navbar .avatar-large:hover,
+#navbar .products:hover {
+  color: #6d28d9;
+  font-weight: "bold";
+}
+
+#navbar .cart .badge-avatar-large-icons {
+  height: 1.4rem;
+  width: 1.4rem;
+  padding: 0.5rem;
+  border: 0rem solid var(--badge-white-color);
+}
+
+#navbar .avatar-badge-icons-cart {
+  position: absolute;
+  top: 0.9rem;
+  right: 0.7rem;
+}
+
+#navbar .badge-red {
+  background-color: var(--badge-white-color);
+  color: var(--header-link-color);
+}
+
+#navbar .avatar-small {
+  height: 2.5rem;
+  width: 2.5rem;
+}
+
+#navbar .products {
+  font-weight: bolder;
+  border-bottom: none;
+}
+
+/* sidebar */
+#side-bar .checkbox {
+  width: 1rem;
+  height: 1rem;
+  border: 1px solid #c3c2c9;
+  background: #fff;
+  border-radius: 1px;
+  margin-right: 0.8rem;
+}
+
+.side-bar {
+  top: 10rem;
+  min-height: 100%;
+}
+
+#side-bar ul {
+  padding: 0rem;
+}
+
+#side-bar ul > li .checkbox-label {
+  display: inline-flex;
+  justify-content: space-between;
+  align-items: center;
+  cursor: pointer;
+}
+
+#side-bar ul > li .checkbox-label > input {
+  height: 25px;
+  width: 25px;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  -o-appearance: none;
+  appearance: none;
+  border: 1px solid #8b5cf6;
+  border-radius: 4px;
+  outline: none;
+  transition-duration: 0.3s;
+  background-color: #ddd6fe;
+  cursor: pointer;
+}
+
+#side-bar ul > li .checkbox-label > input:checked {
+  border: 1px solid var(--primary-color);
+  background-color: #8b5cf6;
+}
+
+#side-bar ul > li .checkbox-label > input:checked + span::before {
+  content: "\2713";
+  display: block;
+  text-align: center;
+  color: #fff;
+  position: absolute;
+  left: 1.77rem;
+}
+
+#side-bar ul > li .checkbox-label > input:active {
+  border: 2px solid #8b5cf6;
+}
+
+#side-bar ul > li .price-range-label > input[type="range"] {
+  -webkit-appearance: none;
+  width: 180px;
+  height: 16px;
+  background-color: var(--primary-color);
+  cursor: pointer;
+  overflow: hidden;
+}
+
+#side-bar
+  ul
+  > li
+  .price-range-label
+  > input[type="range"]::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  width: 16px;
+  height: 16px;
+  border-radius: 50%;
+  background: #fff;
+  box-shadow: -205px 0 0 200px #8b5cf6;
+}
+
+#side-bar .side-bar-box {
+  width: 15rem;
+  height: 3rem;
+  position: fixed;
+  top: 7rem;
+  left: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.8rem;
+  border: 0.01rem solid var(--primary-color);
+}
+
+#side-bar .side-bar-box .btn {
+  padding: 0rem;
+  letter-spacing: 0;
+}
+
+#side-bar .side-bar-box .bg-red {
+  background-color: var(--badge-white-color);
+  color: #7c3aed;
+}
+
+#side-bar .side-bar-box .bg-red:focus {
+  background: none;
+}
+
+/* main */
+
+.content .sort {
+  position: absolute;
```

Comment:
`position: absolute` is causing the sort component to overlap with the product cards in the product-listing page. Consider changing the layout here.